src/SFA.DAS.AdminService.Web/Controllers/Roatp/Apply/RoatpGatewayController.cs

```
@@ -244,16 +251,46 @@ public async Task<IActionResult> EvaluateOrganisationStatus(OrganisationStatusVi
 
-        [HttpGet("/Roatp/Gateway/{applicationId}/Page/1-30")]
-        public async Task<IActionResult> GetOrganisationStatus(Guid applicationId)
+        #region 1.4 - AddressCheck
+        [HttpGet("/Roatp/Gateway/{applicationId}/Page/1-40")]
+        public async Task<IActionResult> GetAddressCheck(Guid applicationId)
         {
             var username = _contextAccessor.HttpContext.User.UserDisplayName();
-            return View("~/Views/Roatp/Apply/Gateway/pages/OrganisationStatus.cshtml", await _mediator.Send(new GetOrganisationStatusRequest(applicationId, username)));
+            return View("~/Views/Roatp/Apply/Gateway/pages/AddressCheck.cshtml", await _mediator.Send(new GetAddressRequest(applicationId, username)));
         }
 
+        [HttpPost("/Roatp/Gateway/{applicationId}/Page/1-40")]
+        public async Task<IActionResult> EvaluateAddressCheck(AddressCheckViewModel viewModel)
+        {
+            var username = _contextAccessor.HttpContext.User.UserDisplayName();
+
+            var validationResponse = await _gatewayValidator.Validate(viewModel);
+            if (validationResponse.Errors != null && validationResponse.Errors.Any())
+            {
+                viewModel.ErrorMessages = validationResponse.Errors;
+                return View("~/Views/Roatp/Apply/Gateway/pages/AddressCheck.cshtml", viewModel);
+            }
+
+            SetupGatewayPageOptionTexts(viewModel);
+
+            var pageData = JsonConvert.SerializeObject(viewModel);
+            _logger.LogInformation($"RoatpGatewayController-EvaluateAddressCheck-SubmitGatewayPageAnswer - ApplicationId '{viewModel.ApplicationId}' - PageId '{viewModel.PageId}' - Status '{viewModel.Status}' - UserName '{username}' - PageData '{pageData}'");
+            try
+            {
+                await _applyApiClient.SubmitGatewayPageAnswer(viewModel.ApplicationId, viewModel.PageId, viewModel.Status, username, pageData);
+            }
+            catch (Exception ex)
```

Is this swallowing the exception?